fireant/widgets/reacttable.py

```diff
@@ -94,7 +115,8 @@ def __init__(
         transpose=False,
         sort=None,
         ascending=None,
-        max_columns=None
+        max_columns=None,
+        formatting_rules=[],
```

Comment:
It's possible this mutable arg will cause code complaints. The pivot arg defaults to an empty tuple.

Reply:
I'll change it to a tuple.